RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -57,18 +57,21 @@
 RecoLocalCaloRECO.outputCommands.extend(ecalLocalRecoRECO.outputCommands)
 RecoLocalCaloAOD.outputCommands.extend(ecalLocalRecoAOD.outputCommands)
 
-def _modifyRecoLocalCaloEventContentForHGCalRECO( obj ):
-    obj.outputCommands.append('keep *_HGCalRecHit_*_*')
+# mods for HGCAL
 
-def _modifyRecoLocalCaloEventContentForHGCalFEVT( obj ):
-    obj.outputCommands.append('keep *_HGCalRecHit_*_*')
-    obj.outputCommands.append('keep *_HGCalUncalibRecHit_*_*')
+_phase2_RecoLocalCaloFEVT_outputCommands = RecoLocalCaloFEVT.outputCommands
+_phase2_RecoLocalCaloFEVT_outputCommands.append('keep *_HGCalRecHit_*_*')
+_phase2_RecoLocalCaloFEVT_outputCommands.append('keep *_HGCalUncalibRecHit_*_*')
```

Comment:
@makortel - Ah, that is probably a shortcoming of my Python knowledge. I'll figure out a way to fix it.